packages/nx/src/command-line/nx-commands.ts

```diff
@@ -1093,7 +1093,10 @@ function parseCSV(args: string[] | string) {
   if (Array.isArray(args)) {
     return args;
   }
-  return args.split(',');
+  const items = args.split(',');
+  return items.map((i) =>
+    i.startsWith('"') && i.endsWith('"') ? i.slice(1, -1) : i
```

Comment:
Should we handle single quotes?

Reply:
Nope - double checked and we only handle double quotes for `nx run` as well - one could probably argue that this should change, but I digress. Lets fix this first, can attack single quotes more holistically later.